File: lcyl-vue/lcyl/src/types/api/code/nurse.ts
import type { PageDomain, BaseEntity } from "../common";

/** 护理员信息配置分页查询参数 */
export interface NurseQueryParams extends PageDomain {
  /** 护理员姓名 */
  name?: string;
  /** 联系电话 */
  phone?: string;
}

/** 护理员信息配置信息 */
export interface Nurse extends BaseEntity {
  /** 主键ID */
  id?: number;
  /** 护理员姓名 */
  name?: string;
  /** 联系电话 */
  phone?: string;
  /** 创建时间 */
  createTime?: string;
}

File: lcyl-java/lcyl/lcyl-code/src/main/java/com/lcyl/code/domain/Nurse.java
package com.lcyl.code.domain;

import org.apache.commons.lang3.builder.ToStringBuilder;
import org.apache.commons.lang3.builder.ToStringStyle;
import com.lcyl.common.annotation.Excel;
import com.lcyl.common.core.domain.BaseEntity;

/**
 * 护理员信息对象 lc_nurse
 * 
 * @author ruoyi
 * @date 2026-03-23
 */
public class Nurse extends BaseEntity
{
    private static final long serialVersionUID = 1L;

    /** 主键ID */
    private Long id;

    /** 护理员姓名 */
    @Excel(name = "护理员姓名")
    private String name;

    /** 联系电话 */
    @Excel(name = "联系电话")
    private String phone;

    public void setId(Long id) 
    {
        this.id = id;
    }

    public Long getId() 
    {
        return id;
    }

    public void setName(String name) 
    {
        this.name = name;
    }

    public String getName() 
    {
        return name;
    }

    public void setPhone(String phone) 
    {
        this.phone = phone;
    }

    public String getPhone() 
    {
        return phone;
    }

    @Override
    public String toString() {
        return new ToStringBuilder(this,ToStringStyle.MULTI_LINE_STYLE)
            .append("id", getId())
            .append("name", getName())
            .append("phone", getPhone())
            .append("createTime", getCreateTime())
            .toString();
    }
}
